### PR TITLE
Fix SSL and make change of region possible

### PR DESCRIPTION
**Pull request by computeralex92**
To address some issues in #243 from issue #242, I made the following changes:

- Add the SSL Certs to the last docker image
- Make the S3 region changeable via a environment variable & a CLI option

### CODE REVIEW

File: cmd/cmd.go [hunk 1]
```diff
@@ -332,7 +338,7 @@ func New() *Cmd {
 				panic("secret-key not set.")
 			} else if bucket := c.String("bucket"); bucket == "" {
 				panic("bucket not set.")
-			} else if storage, err := server.NewS3Storage(accessKey, secretKey, bucket, c.String("s3-endpoint"), logger, c.Bool("s3-no-multipart")); err != nil {
+			} else if storage, err := server.NewS3Storage(accessKey, secretKey, bucket, c.String("s3-endpoint"), logger, c.Bool("s3-no-multipart"), c.String("s3-region")); err != nil {
```
Comment (aspacca) [inline cmd/cmd.go]: note that here `s3-region` can be empty
I would do something like:
```
			region := "eu-west-1"
			if awsRegion := c.String("s3-region"); awsRegion != "" {
				region = awsRegion
			}
```

I set default to `eu-west-1` to not break current installation

the same empty value problem can be for `s3-endpoint`: do you mind testing setting the region but not the endpoint and see if the sdk is smart enough to figure the second?

Comment (computeralex92) [reply cmd/cmd.go]: The SDK is so smart ;-)
If you remove the endpoint and only set the region, it automatically set the correct endpoint in the background.
I would suggest to only remove the default endpoint and let the region default set to eu-west-1.

In the case someone is using e.g. Scaleway they must set region and endpoint to get a success connection to the object storage.

Comment (aspacca) [reply cmd/cmd.go]: so let's go for setting default region only, but snippet should be enough: there's no default endpoint and we leave as it is

Comment (computeralex92) [reply cmd/cmd.go]: With https://github.com/dutchcoders/transfer.sh/pull/244/commits/cb17b4a1ccc5c521b540b5ba1912c86239558d66 the default endpoint is removed; only the region is set with a default value.

**Comment by aspacca** [comment]
I totally forgot about default value from `Cmd`, thanks for resolving in a much more elegant way

**Pull request merged**